javascript/javascript1/week3/NOnoN0nOYes.js

```diff
@@ -0,0 +1,81 @@
+//********** Save a note ************/
+
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({ content: content, id: id });
```

Comment:
When the key and value names are the same, you can define things a bit shorter:
```suggestion
  notes.push({ content, id });
```